integrations/apache/README.md

```diff
@@ -19,6 +19,27 @@ In order to use the Apache Integration it is required to configure
 `apache-config.yml`. Then, depending on your needs, specify all instances that
 you want to monitor with correct arguments.
 
+The integration assumes your `/server-status` is reachable though the next URL:
+```
+http://127.0.0.1/server-status?auto
+```
+
+You can change this URL by modifying the `status_url` instance parameter of your `apache-config.yml` file.
+
+If the `/server-status` is reachable only though an HTTPS connection which uses a certificate that is signed by your own
+Certificate Authority (e.g. for internal use or development purposes), you will need to make sure one of the next actions
+have been taken:
+
+* Install your Certificate Authority in your host. The Apache integration will look for it by default in the host's root
+  bundle for Certificate Authorities.
+* Update the `apache-config.yml` to have at leastone of the next instance parameters configured: `ca_bundle_file` or
+  `ca_bundle_dir`, whose values must be, respectively, the absolute paths to your alternative Certificate Authorities' bundle
+  file (or the directory where they are).
+* Add to the [Infrastructure Agent configuration file](https://docs.newrelic.com/docs/infrastructure/new-relic-infrastructure/configuration/configure-infrastructure-agent))
```

Comment:
Maybe I misunderstand sth. Why this step is needed, why we need to pass this variables to the agent? It's not enough to have them just configured in our config file?

Reply:
You just need only one of the three actions. I have reworded it to make it clear that, if you do one of them, don't need to do the rest.